docs/user-guide/media-editor-window.md

```diff
@@ -0,0 +1,93 @@
+
+* The **Title** of the image
+
+* The **Alternative text**, that appears if the link is broken
+
+* A **Description**, available from the image settings and in preview
+
+* The **Credits**, displayed on the bottom-right of the image
+
+!!! warning
+    **Source** and **Title** are mandatory fields.
+
+With a click on **Save** <img src="../img/button/save-icon.jpg" class="ms-docbutton"/>, the image is insert in images list and immediately available on preview.
+
+<img src="../img/media-editor-window/img-preview.jpg" class="ms-docimage"/>
+
+!!! note
+    It's always possible to change an image's settings through the **Edit** button <img src="../img/button/editing-button.jpg" class="ms-docbutton"/>.
+
+Finally, the image is inserted into the story by clicking on the **Apply** button <img src="../img/button/apply-button2.jpg" class="ms-docbutton"/>.
+
+## Maps
+
+In order to add a map, the user can click on the **Maps** button <img src="../img/button/maps-button.jpg" class="ms-docbutton"/> for swich to the map section. Clicking on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/> and  the **Map Editor** window opens.
+
+<img src="../img/media-editor-window/edit-map.jpg" class="ms-docimage"/>
+
+The tool for costumize the background map are:
+
+* Adding the **Layers** choosing from the <img src="../img/button/catalog-option.jpg" class="ms-docbutton" style="max-height:20px;"/> button in the *Option menu* <img src="../img/button/burger.jpg" class="ms-docbutton"/>. As it is explained in the [Catalog Services](catalog.md).
+
+* Adding  **Annotations** choosing from the <img src="../img/button/annotation-option.jpg" class="ms-docbutton" style="max-height:20px;"/> button in the *Option menu* <img src="../img/button/burger.jpg" class="ms-docbutton"/>. As it is explained in the [Adding Annotations](annotations.md).
+
+* **Import** a map choosing from the <img src="../img/button/import-button.jpg" class="ms-docbutton" style="max-height:20px;"/> button in the *Option menu* <img src="../img/button/burger.jpg" class="ms-docbutton"/>. As it is explained in the [Export and Import files](import.md).
+
+* **Change Background** just as it is explained in the [Basemap switcher](basemap.md)
+
+* **Edit Layers** just clicking on the Layers button <img src="../img/button/show-layers.jpg" class="ms-docbutton"/>  as it is explained in the [Table of Contents](toc.md)
+
+Once the map background is ready the user can proceed with the operation and insert the map *Icon*, the *Title* and the *Description* in the new window that has opened.
+
+<img src="../img/media-editor-window/save-map-backg.jpg" class="ms-docimage"/>
+
+With a click on the **Save** button <img src="../img/button/save_button.jpg" class="ms-docbutton"/> the map is placed in the list of the *Current Story*. The user can now select it and use it as a background map clicking on **Apply** button <img src="../img/button/apply-button2.jpg" class="ms-docbutton"/>.
```

Comment:
With a click on **Save** button <img src="../img/button/save_button.jpg" class="ms-docbutton"/> the map is saved and it will be available in the list of the *Current Story* maps. The story editor can select it to be used as a map component in the story by clicking on **Apply** button <img src="../img/button/apply-button2.jpg" class="ms-docbutton"/>.